function/package.json

```diff
@@ -12,19 +12,16 @@
   "scripts": {
     "lint": "eslint --ignore-path ../.gitignore .",
     "pretest": "eslint --ignore-path ../.gitignore .",
-    "test": "nyc ava --timeout=30s --verbose test/*test.js",
-    "test-report": "mkdir -p test/reports/unit && nyc ava --tap --timeout=30s --verbose test/*test.js > test/reports/unit/ava.tap"
+    "test": "nyc ava --timeout=30s --verbose test/*test.js"
```

Comment:
the `test-report` script was used in Circle test runs, but is no longer used in the GH Actions test runs. Removing this script means we can also remove the `tap-summary` and `tap-xunit` dependencies below.